### PR TITLE
toggle segments

### PR DESCRIPTION
**Pull request by PTLux1517**
Adds menu option for turning body segments on and off

### CODE REVIEW

File: src/views/PopupView.tsx [hunk 1]
```diff
@@ -37,6 +40,9 @@ export default function PopupView(
                     <dt>Controls Help</dt>
                     <dd onMouseOver={()=>setControlsHelpImgNum(0)}>camera</dd>
                     <dd onMouseOver={()=>setControlsHelpImgNum(1)}>selection</dd>
+                    <li><div>Body Segments
```
Comment (vpzomtrrfrt) [inline src/views/PopupView.tsx:43]: `li` elements aren't supposed to be used outside of `ol`, `ul`, or `menu`

File: src/views/RenderView.tsx [hunk 2]
```diff
@@ -239,11 +240,11 @@ export default function RenderView(
                 seg.geometry.attributes.position.setXYZ(0,-startPos.x,startPos.z,startPos.y); //Vicon coord conversion is -x,z,y
                 seg.geometry.attributes.position.setXYZ(1,-endPos.x,endPos.z,endPos.y);
                 seg.geometry.attributes.position.needsUpdate = true;
-                seg.visible = true;
+                seg.visible = showSegments;
```
Comment (vpzomtrrfrt) [inline src/views/RenderView.tsx:243]: if it's not visible, couldn't we skip updating the position too?

Comment (PTLux1517) [reply src/views/RenderView.tsx]: fixed

File: src/views/PopupView.tsx (hunk 1 above)
Comment (PTLux1517) [reply src/views/PopupView.tsx]: fixed

**Review by vpzomtrrfrt — APPROVED** [review]

**Pull request merged**